src/header/header.scss

```diff
@@ -206,6 +207,29 @@
     max-width: 25vw;
     overflow: hidden;
 
+    // Sizes
```

Comment:
I don't think sizes are used anywhere. We default to large size, and automatically reduce height for slim mode

Reply:
I'm talking about the actual size rules below, but you have just removed the comment 😆